.github/workflows/buildsCI.yaml

```diff
@@ -104,7 +104,13 @@ jobs:
       shell: bash
       run: |
         cd ${{env.WIN_BUILD_DIR}}
-        cmake --build . --config ${{env.LRS_BUILD_CONFIG}} -- -m    
+        cmake --build . --config ${{env.LRS_RUN_CONFIG}} -- -m
+
+    - name: LibCI
+      # Build your program with the given configuration
+      shell: bash
+      run: |
+        python3 unit-tests/run-unit-tests.py --no-color --debug --stdout --not-live --context "dds windows" ${{env.WIN_BUILD_DIR}}/Release
```

Comment:
Aren't we missing `-DBUIILD_UNIT_TESTS=true`?
It will cause not run to the CPP tests right?

Reply:
Hmm good thought, not sure but it didn't fail so I didn't even think about it... let me check

Reply:
Seems when we build without `BUILD_UNIT_TESTS`, the manifest doesn't even include them and we simply do not even see the tests... that's why we didn't even try to run them.

So, as it stands, this runs only the python tests. Adding the .exe tests will raise the runtime, but I guess that's acceptable.
I'll add.

Reply:
Thanks,
I also note that you set the DDS context meaning we miss the `nightly` runs.
I guess it meant more to live devices so it's OK..

Reply:
The best experience we can have is same to libCI.
* Prints are not visible on run log
```
Running test x
Test verdict : Pass / Fail
```
* Upload the tests logs
Otherwise it's too much info and the web buffering is bad.. 
We want to focus on failed tests and find it easily

Reply:
Don't agree about the last one... easy to view the raw log and look for what you need -- downloading a zip is annoying

Reply:
That's not what you did on LibCI..
And it was great.. you see a templated simple view.

When you have an issue you want to see the test output and not 3000 lines of prints

Reply:
Yeah well I don't have a web page here that can link to the failed tests. I've already seen some failed tests, trust me it's not difficult right now to find what you're looking for.

On the other hand, to see a failure and then go to another page to download something, open it, find the file you want, open it in some viewer, delete the download, etc. -- that's annoying. We have enough of that with static analysis.